### PR TITLE
Feature/html css skeleton

### PR DESCRIPTION
**Pull request by nalito223**
### Summary: 
This branch includes a starting point for all the html elements and css styling for the site that I will be building in this project. NOTE: this is a static site without any Javascript functionality. 

### How to test:
Pull down the branch and run `NPM start`. Then, explore the site and test if buttons are reactive and that the layout is easily understood while accommodating the user stories. Test the scrolling feature in the table view. Take a look at the CSS and HTML files to see if there is any organizational or syntax issues.

image

### CODE REVIEW

**Comment by AndrewCerveny** [comment]
First off I'm amazed at the static comp and think its a beautiful user friendly design. While looking through it I could picture the user story and knew that things were hard coded only to be inner text || html changed later. 

Index .html page
1) <Welcome John >  should be an h1 tag. since it is the biggest text on the page and the eye grabber h1 is best suited. 
it can be contained in a div. but having the proper tag will also aid is accessibility. 
2) I'd change your books view to an <H2 > more specified header. also can be consisten with the headers later on for total bookings and total spent. 
2) Line [30] the inputs will need to be associated with a form. https://www.w3schools.com/html/html_forms.asp
- tip; you can add a class list hidden to the form label and still pass the requirements for an input being properly associated.
- 3) at the bottom of the index the Total Bookings and Total Spent. can be wrapped in divs but they both should have a proper semantic tag associated I would make these  <h2>

css
1) < stack find a room button and table button> they seem to have the same layout it would looke like 
.stackbutton,
.tablebutton {

}
- don't forget comma between. 
-Side tip please add orgazation comments.  if there is something affecting whole page add it under the wilds* 
for example :  the overlay.
 - if you are grabbing elements by there semantic tag  then have and elements section 
 - main area etc etc. 
 * This tip is mainly for organizational concepts later if you want to improve something or change something it is much easier to reference where in the css the changes can be made. 
 
* all in all great skeleton you have done a lot of hard work already keep trucking!

**Comment by AndrewCerveny** [comment]
I forgot to mention that you hover effect are awesome but depending on the background you might want to add some color variation. the current background is more grey and the hover effects are slightly greyier. Maybe make them blue or fill in the table buttons to be a blue. There needs to be a little more color contrasting for an enjoyable user experience!

**Comment by nalito223** [comment]
Thanks so much for the feedback! I just pushed up changes so I think we're good to go if you are able to take another look.

File: dist/index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
   <div class="main-container">
     <header>
       <nav class="main-nav-container">
-        <div class="nav-container-left">Welcome, John!</div>
+        <div class="nav-container-left" label="Welcome message for user">Welcome, John!</div>
```
Comment (AndrewCerveny) [inline dist/index.html:19]: label works ! nice upgrade.

**Review by AndrewCerveny — CHANGES_REQUESTED** [review]
Index HTML 

1st index html 30 -32 when dealing with form inputs here is a tip and example to get you going. 

<form **for="tag-select"** class="filter-by-tag-display">
          <label class="label-for-select-a-tag" **for="tag-select"**
            >Select a tag:</label
          >
          <select **id="tag-select**"class="list-of-tag-options" name="tags">
            <option class="tag-options-text" value="default">
              Please choose a tag
            </option>
          </select>

2) line 28 < make that into an <h2></h2> 
3) change lines 155 and 156 
from  <div>Total bookings: 5</div>
to <div > 
<h2> Total Bookings : 5 </h2>
    </div>

CSS:
saw the stacking nice job! 
please put in comment dividers on the secitons 
!====== main elements !====

!== nav bar  styles !======

!=== main Body Styles !=====

!=== Table Styles !======== 

Comments for organization are perfectly fine. It helps people track where the styles are being applied and will also help you come back to  areas where you might want to add more css functionality. 
Please add some color to your hover css. it makes for a better user experience.

**Review by AndrewCerveny — COMMENTED** [review]
Index HTML 

1st index html 30 -32 when dealing with form inputs here is a tip and example to get you going. 

<form **for="tag-select"** class="filter-by-tag-display">
          <label class="label-for-select-a-tag" **for="tag-select"**
            >Select a tag:</label
          >
          <select **id="tag-select**"class="list-of-tag-options" name="tags">
            <option class="tag-options-text" value="default">
              Please choose a tag
            </option>
          </select>

2) line 28 < make that into an <h2></h2> 
3) change lines 155 and 156 
from  <div>Total bookings: 5</div>
to <div > 
<h2> Total Bookings : 5 </h2>
    </div>

CSS:
saw the stacking nice job! 
please put in comment dividers on the secitons 
!====== main elements !====

!== nav bar  styles !======

!=== main Body Styles !=====

!=== Table Styles !======== 

Comments for organization are perfectly fine. It helps people track where the styles are being applied and will also help you come back to  areas where you might want to add more css functionality. 
Please add some color to your hover css. it makes for a better user experience.

**Review by AndrewCerveny — APPROVED** [review]
We've had a discussion i've sent the changes over via slack we are good to go.

**Comment by nalito223** [comment]
@hfaerber tagging you in this PR that I sent to my Acountabilibuddy.

**Pull request merged**